venv.go

```diff
@@ -86,6 +88,19 @@ func (c VenvCommand) Run() (string, string, error) {
 
 	defer cancel() // The cancel should be deferred so resources are cleaned up
 
+	path, ok := os.LookupEnv("PATH")
+	if !ok {
+		logrus.Error("unable to lookup the $PATH env variable")
```

Comment:
Is this worthy of dropping out of execution?
Currently this is just going to log the error and keep moving on.

Reply:
I feel that it's an issue, later on, we will just have a failure during the execution of the command.